### PR TITLE
Revit/mep networks

### PR DESCRIPTION
**Pull request by golembiovskyi**
## Checklist:

- [ ] My pull request follows the guidelines in the [Contributing guide](https://github.com/specklesystems/speckle-server/blob/main/CONTRIBUTING.md)?
- [ ] My pull request does not duplicate any other open [Pull Requests](../../pulls) for the same update/change?
- [ ] My commits are related to the pull request and do not amend unrelated code or documentation.
- [ ] My code follows a similar style to existing code.
- [ ] I have added appropriate tests.
- [ ] I have updated or added relevant documentation.

### CODE REVIEW

**Review by golembiovskyi — COMMENTED** [review]
golembiovskyi:revit/mep-networks

File: Objects/Converters/ConverterRevit/ConverterRevitShared/Partial Classes/ConvertFamilyInstance.cs [hunk 1]
```diff
@@ -172,34 +172,50 @@ public ApplicationObject FamilyInstanceToNative(BuiltElements.Revit.FamilyInstan
...
-      {
-        if (speckleFi.rotation != (familyInstance.Location as LocationPoint).Rotation)
-        {
-          var axis = DB.Line.CreateBound(new XYZ(basePoint.X, basePoint.Y, 0), new XYZ(basePoint.X, basePoint.Y, 1000));
-          (familyInstance.Location as LocationPoint).Rotate(axis, speckleFi.rotation - (familyInstance.Location as LocationPoint).Rotation);
-        }
-      }
-      catch { }
-
+      #region OLD ROTATION CODE
```
Comment (clairekuang) [inline Objects/Converters/ConverterRevit/ConverterRevitShared/Partial Classes/ConvertFamilyInstance.cs:175]: Why is this region commented out in the FamilyInstance class? Let's not make any changes to this class and fix any rotation problems in a separate issue 0 please remove these changes :)

File: Objects/Converters/ConverterRevit/ConverterRevitShared/Partial Classes/ConvertFamilyInstance.cs [hunk 2]
```diff
@@ -324,6 +340,11 @@ private Base PointBasedFamilyInstanceToSpeckle(DB.FamilyInstance revitFi, Point
       speckleFi.handFlipped = revitFi.HandFlipped;
       speckleFi.level = lev1 != null ? lev1 : lev2;
       speckleFi.mirrored = revitFi.Mirrored;
+      //var transform = revitFi.GetTotalTransform();
```
Comment (clairekuang) [inline Objects/Converters/ConverterRevit/ConverterRevitShared/Partial Classes/ConvertFamilyInstance.cs:343]: Remove these unused lines!

File: Objects/Converters/ConverterRevit/ConverterRevitShared/Partial Classes/ConvertFamilyInstance.cs (hunk 1 above)
Comment (golembiovskyi) [reply Objects/Converters/ConverterRevit/ConverterRevitShared/Partial Classes/ConvertFamilyInstance.cs:175]: go it, you can discard my changes

**Review by clairekuang — APPROVED** [review]

**Pull request merged**